node/peer_score_strategies.js

```diff
@@ -74,3 +76,57 @@ PreferOutgoing.prototype.getScore = function getScore() {
             return 0.4 + random * 0.6;
     }
 };
+
+module.exports.PreferIncoming = PreferIncoming;
+
+function PreferIncoming(peer) {
+    var self = this;
+
+    self.peer = peer;
+    self.lastTier = self.getTier();
+}
+
+PreferIncoming.UNCONNECTED = 0;
+PreferIncoming.ONLY_OUTGOING = 1;
+PreferIncoming.FRESH_INCOMING = 2;
+PreferIncoming.READY_INCOMING = 3;
+
+PreferIncoming.prototype.getTier = function getTier() {
+    var self = this;
+
+    var inconn = self.peer.getInConnection();
+    var outconn = self.peer.getOutConnection();
+
+    if (!inconn && !outconn) {
+        return PreferIncoming.UNCONNECTED;
```

Comment:
Let's favor UNCONNECTED lower then connected.

This leads to bugs where `choosePeer()` picks a `Peer` and then gets an ECONNREFUSED.